test/functional/feature_notifications.py

```diff
@@ -66,6 +77,10 @@ def run_test(self):
             txids_rpc = list(map(lambda t: t['txid'], self.nodes[1].listtransactions("*", block_count)))
             assert_equal(sorted(txids_rpc), sorted(os.listdir(self.walletnotify_dir)))
 
+            self.log.info("test -walletnotify with importmulti")
+            self.test_importmulti(updateexisting=True, count=11)
```

Comment:
suggested clarification
```diff
-            self.test_importmulti(updateexisting=True, count=11)
+            self.test_importmulti(updateexisting=True, count=block_count+1)
```

Reply:
and maybe the 1 in each `self.test_importmulti` call and inside the function with a `NEW_ADDRESSES_COUNT` constant. Ideally the number could vary randomly with each test run.

Reply:
Let's keep this simple for now.